aws/cf.go

```diff
@@ -15,21 +14,23 @@ const (
 	certificateARNTagLegacy = "ingress:certificate-arn"
 	certificateARNTagPrefix = "ingress:certificate-arn/"
 	ingressOwnerTag         = "ingress:owner"
+	cwAlarmConfigHashTag    = "cloudwatch:alarm-config-hash"
 )
 
 // Stack is a simple wrapper around a CloudFormation Stack.
 type Stack struct {
-	Name            string
-	status          string
-	DNSName         string
-	Scheme          string
-	SecurityGroup   string
-	SSLPolicy       string
-	IpAddressType   string
-	TargetGroupARN  string
-	CertificateARNs map[string]time.Time
-	OwnerIngress    string
-	tags            map[string]string
+	Name              string
+	status            string
+	DNSName           string
+	Scheme            string
+	SecurityGroup     string
+	SSLPolicy         string
+	IpAddressType     string
+	TargetGroupARN    string
+	CertificateARNs   map[string]time.Time
+	OwnerIngress      string
+	CWAlarmConfigHash string
```

Comment:
Can you move the 2 string members OwnerIngress and CWAlarmConfigHash just before TargetGroupARN, please?
This will pack the struct better.

Reply:
Sure thing, fixed!